.gitignore

```diff
@@ -6,3 +6,7 @@
 
 # Binaries
 /cmd/healthcheck/healthcheck
+
+# Config files
+.idea
+.editorconfig
```

Comment:
the point of `.editorconfig` is to sync it with every developer so the code has a consistent tabstop (it's also supported by github so when you view the files on github it picks tabstop from `.editorconfig`)

Reply:
Developers should be free to configure their editors however is comfortable for them.
Ensuring that basic standards are maintained in the codebase should be left up to linters.

You could probably convince me to leave the bazel ones in there but I would rather defer to buildifier, which is enforceable, rather than .editorconfig.

Reply:
for go code, even if tab is enforced, the size of the tab actually affects where do people start to wrap the long lines.

Reply:
no one uses `.editorconfig` to config their local editor, if they want to do that they will do that in the editor instead (vimrc, vscode config, etc.)

the whole point of `.editoconfig` is to sync the config across the project.

Reply:
Reverted.